gtclang/src/gtclang/Frontend/ClangASTExprResolver.h

```diff
@@ -18,6 +18,7 @@
 #define GTCLANG_FRONTEND_CLANGASTEXPRRESOLVER_H
 
 #include "dawn/SIR/ASTFwd.h"
+#include "dawn/Support/SourceLocation.h"
```

Comment:
necessary?

Reply:
Yeah, methods defined on lines 244 and 247 take `std::ostream`s.

Reply:
Yeah, methods defined on lines 94 and 95 return `dawn::SourceLocation` objects.